low-level/files/opt/seagate/sspl/conf/kafka.service

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+[Unit]
```

Comment:
These service files for kafka & zookeeper should be with cortx-utils (foundation team), why we are creating & trying to maintain here ?

Reply:
This is just for SSPL standalone installation. Services will only be created when SSPL is deployed using sspl_dev_deploy and these files are only used in sspl_dev_deploy.

Reply:
Done